<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">            QWidget {                                          
            background-color:             rgb(16, 37, 63);
            }                                                  
                                                               
            QPushButton{                                       
            background-color:             rgb(55, 96,146);
            font:14pt 'Segoe UI Semibold';
            color:                        rgb(220,230,242);
            border-width:2px;
            border-color:                 rgb(220,230,242);
            }                                                  
                                                               
            QPushButton:checked{                               
            background-color:             rgb(220,230,242);
            color:                        rgb(55, 96,146);
            }                                                  
                                                               
            QPushButton:disabled {                             
            color:                        rgb(242,242,242);
            background-color:             rgb(166,166,166);
            border-color:                 rgb(166,166,166);
            }                                                  
                                                               
            QLabel{                                            
            font:14pt 'Segoe UI Semibold';
            color:                        rgb(220,230,242);
            text-align:center;                                 
            border-style:none;                                 
            }                                                  
                                                               
            QGroupBox{                                         
            + largeFont +
            color:                        rgb(220,230,242);
            text-align:center;                                 
            }                                                  
                                                               
            QComboBox {                                        
            border-style:none;                                 
            color:                        rgb(220,230,242);
            background-color:             rgb(55, 96,146);
            font:14pt 'Segoe UI Semibold';
            }                                                  
            QComboBox:disabled {                               
            color:                        rgb(242,242,242);
            background-color:             rgb(166,166,166);
            }                                                  
                                                               
            QComboBox QAbstractItemView {                      
            color:                        rgb(220,230,242);
            background-color:             rgb(55, 96,146);
            selection-color:              rgb(55, 96,146);
            selection-background-color:   rgb(220,230,242);
            }												    
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QWidget" name="wMinionHubContainer" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblMinionHub">
         <property name="text">
          <string>Idle Minions</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="VerticalScrollArea" name="saMinionHub">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="MinionHub" name="wMinionHub">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>184</width>
            <height>110</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="wQuestHubContainer" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="VerticalScrollArea" name="saQuestHub">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QuestHub" name="wQuestHub">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>183</width>
            <height>110</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblQuestHub">
         <property name="text">
          <string>Available Quests</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QWidget" name="wQuestProgressHubContainer" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="VerticalScrollArea" name="saQuestProgressHub">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QuestProgressHub" name="wQuestProgressHub">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>543</width>
            <height>110</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblQuestProgressHub">
         <property name="text">
          <string>Quests in Progress</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pbGo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Perform Quest</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QWidget" name="wTitle" native="true">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="lblGoldText">
         <property name="text">
          <string>Available Gold</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblGold">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QuestHub</class>
   <extends>QWidget</extends>
   <header>questhub.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MinionHub</class>
   <extends>QWidget</extends>
   <header>minionhub.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QuestProgressHub</class>
   <extends>QWidget</extends>
   <header>questprogresshub.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VerticalScrollArea</class>
   <extends>QScrollArea</extends>
   <header>verticalscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
